extras/videoDrivers/SDL/VideoSDL.cpp

```diff
@@ -45,7 +47,29 @@ struct DeleterDeleteRC
     void operator()(HGLRC rc) const { wglDeleteContext(rc); }
 };
 
+SDL_SysWMinfo GetWMInfo()
+{
+    SDL_SysWMinfo info;
+    SDL_VERSION(&info.version);
+    const int result = SDL_GetWMInfo(&info);
+    if(result == 1)
+        return info;
+    throw makeException("GetWMInfo failed with ", result);
+}
+auto CreateOGLContext(HDC deviceContext)
+{
+    std::unique_ptr<HGLRC, DeleterDeleteRC> context{wglCreateContext(deviceContext)};
+    if(!context)
+        throw makeLastSystemError("Failed to create OGL context");
+    return context;
+}
+void ShareOGLResources(HGLRC srcContext, HGLRC dstContext)
+{
+    if(!wglShareLists(srcContext, dstContext))
+        throw makeLastSystemError("Failed to create OGL context");
```

Comment:
copy and paste. I think here there should be a "Failed to share OGL lists".

Reply:
Yes thx. We do have a problem though: We need Boost build with fPIC if we want to use it in the drivers. For the purpose of `makeLastSystemError` I could implement that myself, but if we need more of boost we need the fPIC or shared boost libs.

Reply:
Fixing this by using std::error_code